### PR TITLE
MSTORE8 opcode spec

### PR DESCRIPTION
**Pull request by Brechtpd**
Implementation: https://github.com/appliedzkp/zkevm-circuits/pull/160

### CODE REVIEW

File: specs/opcode/51MLOAD_52MSTORE_53MSTORE8.md [hunk 1]
```diff
@@ -24,55 +26,63 @@ memory_cost(memory_size) := Gmem * memory_size + floor(memory_size * memory_size
 
 The gas cost charged for the op is the additional `memory_size` needed:
 ```
-next_memory_size := max(curr_memory_size, memory_size(address + 32))
+MLOAD/MSTORE: offset := 32, MSTORE8 := offset := 1
```
Comment (han0110) [inline specs/opcode/51MLOAD_52MSTORE_53MSTORE8.md:29]:
```suggestion
MLOAD/MSTORE: offset := 32, MSTORE8: offset := 1
```

**Review by han0110 — APPROVED** [review]
LGTM!

**Review by miha-stopar — APPROVED** [review]
LGTM! Will merge once the formatting nitpick is fixed.

File: specs/opcode/51MLOAD_52MSTORE_53MSTORE8.md (hunk 1 above)
Comment (Brechtpd) [reply specs/opcode/51MLOAD_52MSTORE_53MSTORE8.md]: Done, thanks!

**Pull request merged**